_posts/2020-09-21-changelog-131.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #131"
+date: 2020-09-17 12:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6039 -->
+* Provide intention to create a method from an unresolved method call (by [@Kobzol])
+
+    {% include gif-img.html path="/assets/posts/changelog-131/create_method" w="650px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5928 -->
+* Add initial parsing of [the new asm! macro](https://github.com/rust-lang/rfcs/pull/2873) (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5939 -->
+<!-- * RUN: Add build task for wasm-pack configurations -->
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5720 -->
+* Refactor checking and filling of match expressions
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5369 -->
+* Deprioritize completion of `&mut self` methods for const references and bindings (by [@Kobzol])
```

Comment:
Maybe something like "Place methods with `&mut self` at the end of completion list for non-mutable variables"?
Just to make it clearer